libs/localization/assets/locales/en/char_Dehya.json

```diff
@@ -3,6 +3,5 @@
   "field_dmg_": "Fiery Sanctum DMG Bonus",
   "initialHeal": "Initial Healing",
   "dotHeal": "Additional Healing",
-  "critHitDuringBurst": "Number of CRIT hits during a single use of <strong>The Lioness' Bite</strong>",
-  "onHitHeal": "On-hit Healing"
```

Comment:
Why are we removing this? we dont have an occurrence of this string in the system.

Reply:
More vague doesn't mean worse. It's standardizing.